stylesheet.scss

```diff
@@ -89,7 +89,7 @@ StEntry.search-entry {
 #searchResults {
   background-color: rgba(255,255,255,0.92);
   border-radius: 4px;
-  max-width: 875px;
+  width: 875px;
```

Comment:
hmm, what if the resolution is lower than that? do we have a minimum supported resolution?

Reply:
I've reworked search to avoid a few other edge cases, and implement one overlooked design aspect of the search view.